src/model/entity/DraftEntity.js

```diff
@@ -37,7 +35,8 @@ var instanceKey = 0;
  * generated via DraftEntity.create() and used to obtain entity metadata
  * via DraftEntity.get().
  */
-var DraftEntity = {
+const DraftEntity = {
```

Comment:
I think that if entities are being moved to `ContentState`, we can eliminate this module altogether -- all creating/updating behavior can be methods on `ContentState`.